pkg/webhook/server/generic/pod/opslifecycle/validating.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 func (lc *OpsLifecycle) Validating(ctx context.Context, c client.Client, oldPod, newPod *corev1.Pod, operation admissionv1.Operation) error {
-	if !utils.ControlledByKusionStack(newPod) {
+	if operation == admissionv1.Delete || !utils.ControlledByKusionStack(newPod) {
```

Comment:
Add a comment for later reading?

Reply:
The following annotations were added.
> 	// GraceDeleteWebhook FeatureGate defaults to false
	// Add '--feature-gates=GraceDeleteWebhook=true' to container args, to enable gracedelete webhook